docs/getting-started.md

```diff
@@ -15,7 +15,7 @@ Download the `containerd-<VERSION>-<OS>-<ARCH>.tar.gz` archive from https://gith
 verify its sha256sum, and extract it under `/usr/local`:
 
 ```console
-$ tar Cxzvf /usr/local containerd-1.6.2-linux-amd64.tar.gz
+$ tar -xzvf /usr/local containerd-1.6.2-linux-amd64.tar.gz
```

Comment:
Doesn't seem typo

Reply:
I mean, the arguments of ``tar`` will not work this way.

Reply:
my fault sorry @AkihiroSuda

Reply:
`Cxzvf` works, while `-xzvf` does not:

```console
# tar Cxzvf /usr/local containerd-1.6.2-linux-amd64.tar.gz
bin/
bin/containerd-shim-runc-v2
bin/containerd-shim
bin/ctr
bin/containerd-shim-runc-v1
bin/containerd
bin/containerd-stress
```

```console
# tar -xzvf /usr/local containerd-1.6.2-linux-amd64.tar.gz
tar (child): /usr/local: Cannot read: Is a directory
tar (child): At beginning of tape, quitting now
tar (child): Error is not recoverable: exiting now

gzip: stdin: unexpected end of file
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

Reply:
You are right. Appologize for my not knowing enough about ``tar`` command.